tests/models/trillsson_efficient/test_modeling_trillsson_efficient.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch Trillsson_efficient model. """
+
+import unittest
+
+from datasets import load_dataset
+
+from transformers.testing_utils import is_torch_available, require_torch, slow, torch_device
+
+
+if is_torch_available():
+    import torch
+
+    from transformers.models.trillsson_efficient import Trillsson_efficientFeatureExtractor, Trillsson_efficientModel
```

Comment:
```suggestion
    from transformers import Trillsson_efficientFeatureExtractor, Trillsson_efficientModel
```
As it can be seen from the Circle CI tests :

```
==================================== ERRORS ====================================
_ ERROR collecting tests/models/trillsson_efficient/test_modeling_trillsson_efficient.py _
ImportError while importing test module '/home/circleci/transformers/tests/models/trillsson_efficient/test_modeling_trillsson_efficient.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../.pyenv/versions/3.7.12/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/models/trillsson_efficient/test_modeling_trillsson_efficient.py:27: in <module>
    from transformers.models.trillsson_efficient import Trillsson_efficientFeatureExtractor, Trillsson_efficientModel
E   ImportError: cannot import name 'Trillsson_efficientFeatureExtractor' from 'transformers.models.trillsson_efficient' (/home/circleci/transformers/src/transformers/models/trillsson_efficient/__init__.py)
_______________ ERROR collecting tests/utils/test_file_utils.py ________________
tests/utils/test_file_utils.py:23: in <module>
    from transformers import *  # noqa F406
src/transformers/utils/import_utils.py:1077: in __getattr__
    value = getattr(module, name)
src/transformers/utils/import_utils.py:1079: in __getattr__
    raise AttributeError(f"module {self.__name__} has no attribute {name}")
E   AttributeError: module transformers.models.trillsson_efficient has no attribute Trillsson_efficientFeatureExtractor
=============================== warnings summary ===============================
```